packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2357,17 +2391,31 @@ export default class Meeting extends StatelessWebexPlugin {
       }
     }
 
+    let joinObj;
+
     return MeetingUtil.joinMeetingOptions(this, options).then((join) => {
       this.meetingFiniteStateMachine.join();
       LoggerProxy.logger.log('Meeting:index#join --> Success');
 
+      joinObj = join;
+
       return join;
-    }).catch((error) => {
-      this.meetingFiniteStateMachine.fail(error);
-      LoggerProxy.logger.error('Meeting:index#join --> Failed', error);
+    })
+      .then(() => {
+        joinSuccess(joinObj);
+        this.deferJoin = undefined;
```

Comment:
```suggestion
      .then((join) => {
        joinSuccess(join);
        this.deferJoin = undefined;
```
You can remove the `joinObj` and get it from the previous promise's return value

Reply:
that works, doing this now.